src/stylesheets/main.css

```diff
@@ -433,17 +467,73 @@ footer a:hover {
     clip-path: url(#path-contact-end);
   }
 
+  .clipped-stories-contact {
+    -webkit-clip-path: url(#path-stories-contact);
+    clip-path: url(#path-stories-contact);
+  }
+
+  .clipped-stories-impact {
+    -webkit-clip-path: url(#path-stories-impact);
+    clip-path: url(#path-stories-impact);
+  }
+
+  .clipped-home-stat-1 {
+    -webkit-clip-path: url(#path-home-stat-1);
+    clip-path: url(#path-home-stat-1);
+  }
+
+  .clipped-home-stat-2 {
+    -webkit-clip-path: url(#path-home-stat-2);
+    clip-path: url(#path-home-stat-2);
+  }
+
+  .clipped-home-stat-3 {
+    -webkit-clip-path: url(#path-home-stat-3);
+    clip-path: url(#path-home-stat-3);
+  }
+
   figure.quote blockquote {
     font-size: calc(24rem / 16);
   }
 
   small,
   .small {
-    font-size: calc(12rem / 16);
+    font-size: calc(12rem / 16) !important;
+  }
+
+  .stories-map {
+    height: 150px;
+  }
+
+  .impact {
+    z-index: 1025;
+    position: absolute;
+    left: 0;
+  }
+
+  .monterey-and-contact,
+  .absolute-footer {
+    position: relative;
+    top: 620px; /* Height of the absolutely-positioned Impact section */
+  }

```

Comment:
This 620px is the height of the `Our impact` section. Since the `Our impact` section is absolutely positioned and placed vertically between other divs - and therefore taken out of the normal flow, the Monterey-Salinas Transit/Contact section and the footer will be 620 pixels up higher than it should be. This code will add 620px above each section so that the sections will be vertically positioned correctly.

Reply:
It's just the height of the entire parent div (includes the image of the train and the rhombus plus the caption below image): 
<img width="496" alt="image" src="https://github.com/user-attachments/assets/e40ec817-36b6-4f92-bad2-0595ada85bdf">

So there isn't a great way to calculate it. I can turn it into a variable just to make it more readable, but it won't be calculated in anyway. This was the only way we can get an extra-wide div in the row _while_ preserving the sidebar table of contents position.

Reply:
All good, thanks for the explanation!